.circleci/config.yml

```diff
@@ -232,7 +232,15 @@ commands:
           command: |
             setup-kube-auth
             kubectl config use-context << parameters.env >>
-
+      - run:
+          name: run db migrations
+          command: |
+            set +e
```

Comment:
I know this is pedantic, but is there a long form name for this. Like `set -o errexit` or something. Not a blocking comment at all.

Reply:
yeah, fair point on the verbosity :) 
It will still look ugly as CircleCI call the shebang with `#!/bin/bash -eo pipefail`  and I'm overriding it next.